tests/results/ngf-upgrade/edge/edge-oss.md

```diff
@@ -7,39 +7,39 @@ NGINX Plus: false
 GKE Cluster:
 
 - Node count: 12
-- k8s version: v1.29.4-gke.1043004
+- k8s version: v1.29.6-gke.1038001
 - vCPUs per node: 16
 - RAM per node: 65855012Ki
 - Max pods per node: 110
-- Zone: us-east1-b
+- Zone: us-west1-b
 - Instance Type: n2d-standard-16
 
-## Test: Send http /coffee traffic
+## Test: Send https /tea traffic
```

Comment:
why does this happen?

Reply:
These two tests to each endpoint run at the same time, so they may be written back in different orders.